ice/settings.py

```diff
@@ -18,6 +18,10 @@ class Settings(BaseSettings):
     OUGHT_INFERENCE_URL: AnyHttpUrl = "https://inference.elicit.org"
     GOLD_STANDARDS_CSV_PATH: str = "gold_standards/gold_standards.csv"
     GS_QUOTE_FOUND_THRESHOLD: float = 0.75
+    OUGHT_ICE_HOST: str = "0.0.0.0"
+    OUGHT_ICE_PORT: int = 8935
+    OUGHT_ICE_NO_START_SERVER: bool = False
+    OUGHT_ICE_NO_BROWSER: bool = False
```

Comment:
The double-negative is a bit confusing here, how about we change the names and default them to True?